### DISCUSSION TITLE
Prevent PRs from updating across projects

### DISCUSSION THREAD

**Discussion by mattfirtion** [renovatebot/renovate#22321] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.96

### If you're self-hosting Renovate, select which platform you are using.

None

### If you're self-hosting Renovate, tell us what version of the platform you run.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Package updates should not be bundled across projects.

I'm working in a monorepo with several projects.  Renovate may update package `foo` in `projectA` and `projectB`, but the update only works in `projectA`.  Is it possible to have two PRs created, one for each project?

### What you tried so far.

N/A

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
I suspect your question is the same as https://github.com/renovatebot/renovate/discussions/20514
